packages/adapter-mongodb/src/index.ts

```diff
@@ -90,6 +67,7 @@ const adapter = (url: string): Adapter => {
                     ...data.user_data,
                 });
                 await userDoc.save();
+                console.log(await User.find().lean());
```

Comment:
This probably should've been removed, right?

Reply:
Resolved in the recent PR